packages/datadog_gql_link/test/datadog_gql_link_test.dart

```diff
@@ -728,10 +728,11 @@ query UserInfo($id: ID!) {
         final capturedAttrs = verify(
                 () => mockRum.startResource(any(), any(), any(), captureAny()))
             .captured[0] as Map<String, dynamic>;
-        var traceId =
-            BigInt.parse(capturedAttrs[DatadogRumPlatformAttributeKey.traceID]);
+        var traceId = BigInt.parse(
+            capturedAttrs[DatadogRumPlatformAttributeKey.traceID],
+            radix: 16);
         expect(traceId, isNotNull);
-        expect(traceId.bitLength, lessThanOrEqualTo(63));
+        expect(traceId.bitLength, lessThanOrEqualTo(127));
```

Comment:
/question
why 127? Is it because of the reserved sign bit? I'm curious if you use two BitInt to hold lower and higher part, would it actually make 100% compliant?

The problem here, the SDKs will send full 128 bits and we will always lose one bit.

Reply:
🤔 I thought I saw that the top bit on the trace id needed to be zero? I must be mistaken.  I'll change it to allow the full 128 bits.